Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp

```diff
@@ -2816,14 +2818,10 @@ void CanvasRenderingContext2DBase::drawText(const String& text, double x, double
 
 void CanvasRenderingContext2DBase::drawTextUnchecked(const TextRun& textRun, double x, double y, bool fill, std::optional<double> maxWidth)
 {
-    auto measureTextRun = [&](const TextRun& textRun) -> std::tuple<float, FontMetrics>  {
-        auto& fontProxy = *this->fontProxy();
-
-        // FIXME: Need to turn off font smoothing.
-        return { fontProxy.width(textRun), fontProxy.metricsOfPrimaryFont() };
-    };
+    RefPtr<FontProxy> fontProxy = FontProxy::create();
```

Comment:
`RefPtr<FontProxy>` -> `Ref`

Also, why are we now constructing a FontProxy here?